CRISPResso2/CRISPRessoShared.py

```diff
@@ -1218,35 +1218,26 @@ def split_interleaved_fastq(fastq_filename, output_filename_r1, output_filename_
 # allele modification functions
 ######
 
-def get_row_around_cut(row, cut_point, offset):
-    cut_idx = row['ref_positions'].index(cut_point)
-    return row['Aligned_Sequence'][cut_idx - offset + 1:cut_idx + offset + 1], row['Reference_Sequence'][
-                                                                               cut_idx - offset + 1:cut_idx + offset + 1], \
-           row['Read_Status'] == 'UNMODIFIED', row['n_deleted'], row['n_inserted'], row['n_mutated'], row['#Reads'], \
-           row['%Reads']
+def get_row_around_cut_asymmetrical(row,cut_point,plot_left,plot_right):
+    cut_idx=row['ref_positions'].index(cut_point)
+    return row['Aligned_Sequence'][cut_idx-plot_left+1:cut_idx+plot_right+1],row['Reference_Sequence'][cut_idx-plot_left+1:cut_idx+plot_right+1],row['Read_Status']=='UNMODIFIED',row['n_deleted'],row['n_inserted'],row['n_mutated'],row['#Reads'], row['%Reads']
 
 
-def get_dataframe_around_cut(df_alleles, cut_point, offset, collapse_by_sequence=True):
+def get_dataframe_around_cut_asymmetrical(df_alleles, cut_point,plot_left,plot_right,collapse_by_sequence=True):
```

Comment:
There is a call to `get_dataframe_around_cut` in `scripts/plotAmbiguous.py`, could you please update the signature and name there so that it doesn't break?